TrustSDK/Classes/Client/Types/TrustCommand.swift

```diff
@@ -10,11 +10,73 @@ import TrustWalletCore
 enum CommandName: String {
     case getAccounts = "sdk_get_accounts"
     case sign = "sdk_sign"
```

Comment:
can we use `sdk_sign` but make optional param as `send: Bool` here? i don't think need comment is needed here